documentation/step-by-step-install.md

```diff
@@ -0,0 +1,148 @@
+# Step by step install steps and example running the python script
+
+## Get the code
+
+<pre>
+git clone https://github.com/CDLUC3/counter-processor.git
+cd counter-processor
+git checkout <em>branch-or-tag</em>
+</pre>
+
+## Set an environment for python besides your system environment
+<pre>
+python3.6 -m venv counter-env
```

Comment:
Assumes python3.6 is installed, which is not the case on most machines.

Reply:
After running this, must also
`source counter-env/bin/activate`

Reply:
Added this to the instructions.